src/Voron/Data/Fixed/FixedSizeTree.cs

```diff
@@ -319,9 +319,19 @@ private void ThrowInvalidFixedSizeTreeType()
             }
         }
 
+        public void ForceValidateTree()
+        {
+            AssertTreeIsValid();
+        }
+
         [Conditional("VALIDATE")]
         private void ValidateTree()
         {
+            AssertTreeIsValid();    
+        }
+        
+        private void AssertTreeIsValid()
```

Comment:
Why do we need both `ForceValidateTree` and `AssertTreeIsValid` ?

Reply:
We need the "normal", attribute protected path, and we need the calls for the schema update

Reply:
See my changes on top of yours